scripts/build.py

```diff
@@ -395,12 +528,15 @@ def extract(self):
         patches = self.patches
         os.chmod(runfile, 0o777)
         with tempdir() as tmpd:
-            check_call([os.path.join(self.src_dir, runfile),
-                        '--toolkitpath', tmpd, '--toolkit', '--silent'])
+            cmd = [os.path.join(self.src_dir, runfile),
+                        '--toolkitpath', tmpd, '--toolkit', '--no-drm',
```

Comment:
Going by the error on the build farm, it would appear that `CUDA 8.0` doesn't have the `--no-drm` flag, this causes an error during conda build and stops the build job. I guess we could have `extra_flags` or similar as a member of the config dict which then gets added here?